client/src/utils.rs

```diff
@@ -116,6 +122,15 @@ impl Client {
         hex_to_bech32(hex, &bech32_hrp)
     }
 
+    /// Transforms an alias id to a bech32 encoded address
+    pub async fn alias_id_to_bech32(&self, alias_id: AliasId, bech32_hrp: Option<&str>) -> crate::Result<String> {
+        let bech32_hrp = match bech32_hrp {
+            Some(hrp) => hrp.into(),
+            None => self.get_bech32_hrp().await?,
+        };
+        Ok(alias_id_to_bech32(alias_id, &bech32_hrp))
```

Comment:
If you call this in both branches, you avoid the allocation of the `Some` branch. You can change that in the other methods as well.